ui/ui/config_ui.py

```diff
@@ -18,9 +19,31 @@ def show_page_content(self):
 
         st.markdown(f"""
                     # ARISE configuration landing page
-                    
-                    ## The UI configuration is:  
                     """)
+        st.divider
+        # allows the user to configure the base path for the prediction model
+        with st.form(key='config-form'):
+            st.subheader("Configure the base path of the prediction model")
+            st.text_input("Base path", key="base_path", value=get_config("job", "base_path"),
+                          help="The base path of the application prediction model")
+            st.form_submit_button("Save")
```

Comment:
need to add a browse button so user can select the input path rather than just typing it

Reply:
Staged to following PR, new issue here: https://github.com/arise-insights/arise-predictions/issues/18